src/components/YourFeed.tsx

```diff
@@ -224,33 +202,33 @@ export class YourFeed extends React.PureComponent<DispatchProps & StateProps, Yo
         }
     }
 
-    private async unfollowFeed(feed: Feed) {
+    private unfollowFeed = async (feed: Feed) => {
         const confirmUnfollow = await AreYouSureDialog.show('Are you sure you want to unfollow?');
         if (confirmUnfollow) {
             this.props.onUnfollowFeed(feed);
         }
     }
 
-    private followFeed(author: Author) {
+    private followFeed = (author: Author) => {
         const knownFeed = this.props.knownFeeds.find(feed => feed.feedUrl === author.uri);
         if (knownFeed != null) {
             this.props.onFollowFeed(knownFeed);
         }
     }
 
-    private async openPost(post: Post) {
+    private openPost = async (post: Post) => {
         if (post.link) {
             if (!isSwarmLink(post.link)) {
                 await Linking.openURL(post.link);
             }
         }
     }
 
-    private isPostSelected(post: Post) {
-        return this.state.selectedPost && this.state.selectedPost._id === post._id;
+    private isPostSelected = (post: Post): boolean => {
+        return !!this.state.selectedPost && this.state.selectedPost._id === post._id;
```

Comment:
Is this a new convention that we are using instead of `!= null` now?
The old code wasn't conforming either, but this is new to me too.

Reply:
no, this was accidental here. The only time when I intent to use it is with optional boolean fields.